packages/bot/src/scratch/utils.js

```diff
@@ -300,3 +300,24 @@ export const scrollWorkspace = (workspace, scroll_amount, is_horizontal, is_chro
     workspace.scrollbar.set(scroll_x, scroll_y);
 };
 
+export const checkDisabledBlocks = (workspace, event) => {
```

Comment:
@aaron-binary i think the name can be `updateDisableBlock or setDisableBlock` cause your changing the status of blocks here. nice job though 👍